packages/tables/resources/views/index.blade.php

```diff
@@ -258,7 +258,7 @@ class="fi-ta-header-toolbar flex items-center justify-between gap-x-4 px-4 py-3
                 wire:poll.{{ $pollingInterval }}
             @endif
             @class([
-                'fi-ta-content relative divide-y divide-gray-200 overflow-x-auto dark:divide-white/10 dark:border-t-white/10',
+                'fi-ta-content relative divide-y divide-gray-200 overflow-x-auto dark:divide-white/10 dark:border-t-white/10 [&_.fi-ta-cell-content::after]:bg-gray-300 dark:[&_.fi-ta-cell-content::after]:bg-gray-700',
```

Comment:
We should either keep this styling in the Blade view, or move everything to the CSS file.

Reply:
We cannot move this to the CSS file as it needs to be compiled with the user's Tailwind config

Reply:
We can still use `@apply`, right?

Reply:
Not for colors, this CSS file is compiled and shipped, not imported into the user's own CSS file

Reply:
Oh, is this a separate CSS file? Is is because of support for standalone tables?